test/llvm/llvm-invariant/function_local_const.chpl

```diff
@@ -0,0 +1,21 @@
+// This test checks if llvm.invariant.start is added to function local constants
+// Normally primitive type constants could be used, but chapel compiler
+// performs some constant propagation, which doesn't happen in case of records.
+
+record A
+{
+  var a : int;
+}
```

Comment:
@coodie - right, so in other words:
 1. we'll need to use another approach to put llvm.invariant.start after these `init` calls
 2. the current approach just leaves this case alone (and in particular doesn't generate incorrect IR for it)